src/pages/components/header/menu/Menu.tsx

```diff
@@ -22,9 +32,10 @@ function Menu({ pages }: { pages: MenuPage[] }) {
           onClick={() =>
             page.key[0] === '/' ? navigate(page.key) : window.open(page.key, '_blank')
           }>
-          {page.label}
+          {t(page.label)}
         </li>
       ))}
+      <button onClick={handleChangeLanguage}>Change Language</button>
```

Comment:
let's hide this button for now, so we'll be able to merge it within hacktoberfest
```suggestion
      { null && <button onClick={handleChangeLanguage}>Change Language</button>}
```

Reply:
Hi @NoamGaash. Sorry for the late reply. Made the changes just now.